wdl/RecalibrateGq.wdl

```diff
@@ -0,0 +1,105 @@
+version development
```

Comment:
I got rid of the Map, and that allowed switching back to 1.0. It was actually more difficult than I expected. I pass these data to the overlaps code in the form of a json, so I had to make that json in bash, rather than using the cromwell built-in. It's all good now though.